### PR TITLE
improve classical operations docs in manual

### PR DESCRIPTION
**Pull request by ss2165**
just looking at `manual_circuit.rst` should be enough

merge after 1.3 release

### CODE REVIEW

File: docs/manual/_sources/manual_circuit.rst.txt [hunk 1]
```diff
@@ -882,21 +882,21 @@ The primary use for tableaux in ``pytket`` is as a scalable means of specifying
...
+====================================
 
 .. Performing gates conditional on some classical value or expressions
 
 Moving beyond toy circuit examples, many applications of quantum computing require looking at circuits as POVMs for extra expressivity, or introducing error-correcting schemes to reduce the effective noise. Each of these requires performing measurements mid-circuit and then performing subsequent gates conditional on the classical value of the measurement result.
 
 .. Specified by the `condition` kwarg when adding a gate
 
-Any ``pytket`` gate can be made conditional at the point of adding it to the :py:class:`Circuit` by providing the ``condition`` kwarg. The interpretation of ``circ.G(q, condition=reg[0])`` is: "if the  bit ``[reg[0]`` is set, then perform ``G(q)``".
+Any ``pytket`` operation can be made conditional at the point of adding it to the :py:class:`Circuit` by providing the ``condition`` kwarg. The interpretation of ``circ.G(q, condition=reg[0])`` is: "if the  bit ``reg[0]`` is set, then perform ``G(q)``".
```
Comment (CalMacCQ) [inline docs/manual/_sources/manual_circuit.rst.txt:894]: Could you clarify what is meant by the bit reg[0] being set? Does this mean that the operation is performed if reg[0] is assigned a bit value regardless of whether its 0 or 1?

File: docs/manual/_sources/manual_circuit.rst.txt [hunk 2]
```diff
@@ -951,21 +951,54 @@ See the `API reference <classical.html>`_ for more on the possible expressions a
...
+    circ.add_classicalexpbox_register(reg_a ^ reg_b, reg_c)
+    # c[0] = a[1] & b[2]
+    circ.add_classicalexpbox_bit(reg_a[1] & reg_b[2], [reg_c[0]])
+    # if (a > 1) b = 3
+    circ.add_c_setreg(3, reg_b, condition=reg_gt(reg_a, 1))
+
+    # Register arithmetic
+
+    # c = a + b // c (note the use of the floor divide symbol)
+    circ.add_classicalexpbox_register(reg_a + reg_b // reg_c, reg_c)
```
Comment (CalMacCQ) [inline docs/manual/_sources/manual_circuit.rst.txt:979]: I'm not clear on what it means for `add_classicalexpbox_register` to take multiple arguments? Is this new?

**Review by CalMacCQ — COMMENTED** [review]
There are a few things I don't understand here but otherwise looks good. Maybe this manual entry would be more readable if the large code cell was broken up into two or three smaller cells and had some text explanation in between.

File: docs/manual/_sources/manual_circuit.rst.txt (hunk 1 above)
Comment (ss2165) [reply docs/manual/_sources/manual_circuit.rst.txt:894]: it means set to 1, I will clarify

File: docs/manual/_sources/manual_circuit.rst.txt [hunk 3]
```diff
@@ -951,21 +951,54 @@ See the `API reference <classical.html>`_ for more on the possible expressions a
     # if (reg_a >= 3)
     circ.T(qreg[7], condition=reg_geq(reg_a, 3))
     # compound register expressions
-    big_reg_exp = reg_a & reg_b | reg_c
+    big_reg_exp = (reg_a & reg_b) | reg_c
```
[the remark this answers is not in the document]
Comment (ss2165) [reply docs/manual/_sources/manual_circuit.rst.txt:954]: and the values of `reg_a` and `reg_b` together bitwise, then or the result with `reg_c` bitwise

File: docs/manual/_sources/manual_circuit.rst.txt (hunk 2 above)
Comment (ss2165) [reply docs/manual/_sources/manual_circuit.rst.txt:979]: the second argument is the register to write the result of the expression to

File: docs/manual/_sources/manual_circuit.rst.txt [hunk 4]
```diff
@@ -882,21 +882,21 @@ The primary use for tableaux in ``pytket`` is as a scalable means of specifying
...
 
 .. Specified by the `condition` kwarg when adding a gate
 
-Any ``pytket`` gate can be made conditional at the point of adding it to the :py:class:`Circuit` by providing the ``condition`` kwarg. The interpretation of ``circ.G(q, condition=reg[0])`` is: "if the  bit ``[reg[0]`` is set, then perform ``G(q)``".
+Any ``pytket`` operation can be made conditional at the point of adding it to the :py:class:`Circuit` by providing the ``condition`` kwarg. The interpretation of ``circ.G(q, condition=reg[0])`` is: "if the  bit ``reg[0]`` is set, then perform ``G(q)``".
 Conditions on more complicated expressions over the values of `Bit <circuit.html#pytket.circuit.Bit>`_ and `BitRegister <circuit.html#pytket.circuit.BitRegister>`_ are also possible, expressed as conditions on the results of expressions involving bitwise AND (&), OR (|) and XOR (^) operations.
```
[the remark this answers is not in the document]
Comment (ss2165) [reply docs/manual/_sources/manual_circuit.rst.txt:895]: `Bit` and `BitRegister` are wrong, `API reference` is a relative link, which _should_ work in production, but I'm not sure. I'd rather avoid absolute URLs if I can for things under the same domain

**Review by CalMacCQ — APPROVED** [review]

**Pull request merged**